package.json

```diff
@@ -1,14 +1,35 @@
 {
-    "name": "destack-workspace",
-    "version": "1.0.0",
-    "workspaces": [
-        "./lib/*",
-        "./dev/*"
-    ],
-    "scripts": {
-        "start": "concurrently --kill-others \"cd lib/destack && npm run dev\" \"cd dev/nextjs-project && npm run dev\""
-    },
-    "devDependencies": {
-        "concurrently": "^6.2.0"
-    }
+  "name": "destack-workspace",
+  "version": "1.0.0",
+  "workspaces": [
+    "./lib/*",
+    "./dev/*"
+  ],
+  "scripts": {
+    "start": "concurrently --kill-others \"cd lib/destack && npm run dev\" \"cd dev/nextjs-project && npm run dev\"",
+    "prepare": "husky install"
+  },
+  "prettier": {
```

Comment:
Can we remove this as we have a `.prettierrc.js`?

Reply:
Yep!